### ISSUE TITLE
Contribs with Coq 8.10

### ISSUE THREAD

**Issue by clarus** [coq/opam-coq-archive#1063]
Hi,

There are some errors with the contribs for Coq 8.10, namely the followings:
* bdds: https://coq-bench.github.io/clean/Linux-x86_64-4.07.1-2.0.1/released/8.10.2/bdds/8.10.0.html
* graphs: https://coq-bench.github.io/clean/Linux-x86_64-4.07.1-2.0.1/released/8.10.2/graphs/8.10.0.html
* mutual-exclusion: https://coq-bench.github.io/clean/Linux-x86_64-4.07.1-2.0.1/released/8.10.2/mutual-exclusion/8.10.0.html
* reflexive-first-order: https://coq-bench.github.io/clean/Linux-x86_64-4.07.1-2.0.1/released/8.10.2/reflexive-first-order/8.10.0.html
* smc: https://coq-bench.github.io/clean/Linux-x86_64-4.07.1-2.0.1/released/8.10.2/smc/8.10.0.html
* tree-automata: https://coq-bench.github.io/clean/Linux-x86_64-4.07.1-2.0.1/released/8.10.2/tree-automata/8.10.0.html

In particular, for `coq-mutual-exclusion.8.10.0` there is this message:
```
File "./binary/version1/Soundness.v", line 341, characters 0-1134:
- Error: Anomaly "The AST is malformed, found prod without binders."
- Please report at http://coq.inria.fr/bugs/.
```

@herbelin Do you think we should remove these packages waiting for fixes? Or there is a simple way to fix them?

I did not see other errors for 8.10.

**Comment by herbelin** [comment]
The problems for graphs, bdds, smc and tree-automata comes from int-map which I apparently made wrong (the tag was not on the correct source). See #1066 for a fix.

The problem with mutual-exclusion is a pretty ennoying bug of coq 8.10.2 (see https://github.com/coq/coq/pull/11241/commits/eff360695af18476bfca0edfdea467cbf9203984). I should not have prepared the opam package yet.

For reflexive-first-order, the tag was wrong. Also fix with #1066.

**Comment by clarus** [comment]
OK top, thanks!
